components/ui/vector/Remix.tsx

```diff
@@ -74,6 +74,12 @@ function SvgRemix(props: any) {
         <symbol id="remix_svg__ri-search-line" viewBox="0 0 32 32">
           <path d="M24.041 22.156l5.711 5.709-1.887 1.887-5.709-5.711a11.871 11.871 0 01-7.472 2.625h-.018.001c-6.624 0-12-5.376-12-12s5.376-12 12-12 12 5.376 12 12v.017c0 2.838-.99 5.446-2.643 7.495l.018-.023zm-2.674-.989A9.27 9.27 0 0024 14.682v-.015.001a9.33 9.33 0 00-9.333-9.333 9.33 9.33 0 00-9.333 9.333 9.33 9.33 0 009.333 9.333h.015a9.273 9.273 0 006.487-2.635l-.002.002.2-.2z" />
         </symbol>
+        <symbol id="remix_svg__ri-close-large-line" viewBox="0 0 32 32">
+          <path d="M14.115 16 3.724 5.609 5.61 3.723l10.391 10.39 10.391-10.39 1.886 1.886L17.887 16l10.391 10.39-1.886 1.886-10.391-10.391L5.61 28.276 3.724 26.39z" />
+        </symbol>
+        <symbol id="remix_svg__ri-add-fill" viewBox="0 0 32 32">
+          <path d="M14.667 14.667v-8h2.667v8h8v2.667h-8v8h-2.667v-8h-8v-2.667h8z"></path>
+        </symbol>
```

Comment:
What is this for?

Reply:
The way we handle icons 😱

It's before my time, I think we should migrate to something else at some point ... I just followed the existing [instructions](https://github.com/smlxl/evm.codes/tree/8401a1383462e4873c99ea3ceabd09c3cf392005/components/ui/vector) for adding the new icons